src/Spectre.libStatistics.Tests/DifferentiatingFeaturesEstimatorTest.cpp

```diff
@@ -92,13 +92,15 @@ TEST(DifferentiatingFeaturesEstimator, throws_for_inconsistent_number_of_feature
     EXPECT_THROW(estimator.Estimate(narrowDataset, secondDataset), InconsistentNumberOfFeaturesException);
 }
 
+/* disabled because it fails sometimes and needs to be checked
```

Comment:
This way we will forget it as soon, as this gets merged. Check [this](https://stackoverflow.com/a/7208119/3067956). I think that gtest will at least keep spam with messages like: *Hey! You've got 1 disabled test!*.

Reply:
done